service/controller/v15/resource/encryptionkey/resource.go

```diff
@@ -8,81 +8,45 @@ import (
 )
 
 const (
-	// Name is the identifier of the resource.
-	Name = "kmskeyv14"
+	name = "kmskeyv14"
 )
 
-// Config represents the configuration used to create a new cloudformation resource.
 type Config struct {
-	// Dependencies.
 	Encrypter encrypter.Interface
 	Logger    micrologger.Logger
 
-	// Settings.
 	InstallationName string
```

Comment:
Is the installation name used for anything?

Reply:
No. Good catch. Removed.